### PR TITLE
fix old iec link

### PR DESCRIPTION
**Pull request by bturkus**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2333,7 +2333,7 @@ <h3>HDBaseT 8P8C/RJ-45</h3>
 <!-- start Power cables -->
 <div class="well">
   <h2 id="power"><u>Power</u></h2>
-  <p>From Type A to Type N, the <a href="http://www.iec.ch/worldplugs/" target="_blank">IEC website</a> provides descriptions, images, and supporting countries to plugs and sockets used around the world.</p>
+  <p>From Type A to Type N, the <a href="http://www.iec.ch/world-plugs/" target="_blank">IEC website</a> provides descriptions, images, and supporting countries to plugs and sockets used around the world.</p>
```
Comment (retokromer) [inline index.html:2336]: You could also use `https://` rather than `http://`.

Comment (EG-tech) [reply index.html]: added a commit to make explicit, thanks @retokromer!

**Comment by EG-tech** [comment]
good catch, thanks @bturkus!

**Comment by retokromer** [comment]
The updated Brew formula should also be online soon: https://github.com/amiaopensource/homebrew-amiaos/pull/307.

**Pull request merged**